tests/forte/image_annotation_test.py

```diff
@@ -42,6 +43,21 @@ def setUp(self):
         ip.set_cache(self.line)
         ImageAnnotation(self.datapack)
 
+    def test_datapack_image_operation(self):
+        datapack = DataPack("image2")
+        datapack.set_image(self.line, 0)
```

Comment:
looking at these test cases, I am wondering how do we add a next payload? Do we need to create a `add_image` function?

Reply:
set_image can add next payload but having `-1` as the index

Reply:
This usage looks a bit tricky though, and does this behavior applies to all the payloads? We should make them the same.

Reply:
Currently, it applies to payloads of all modalities. I can make another function if that's more user-friendly.

Reply:
yeah, we need to have a natural way to add image, `datapack.set_image(self.line, -1)` is hard to figure out (we also don't have documentation for it).

```
datapack.set_image(image, payload=x) # set the base payload, 
datapack.add_image(image) # add another payload and set the image. This should be the future interface, it starts by assuming there is no image payload in the data pack (i.e. it won't assume there is an index=0 payload).
```